library/ssl_tls.c

```diff
@@ -11119,6 +11119,10 @@ int mbedtls_ssl_read( mbedtls_ssl_context *ssl, unsigned char *buf, size_t len )
     mbedtls_platform_memcpy( buf, ssl->in_offt, n );
     ssl->in_msglen -= n;
 
+    // FI mitigation: "delete" incoming data after it's copied to buffer
+    mbedtls_platform_memset(ssl->in_offt, 0, n);
+    mbedtls_platform_memset(ssl->in_msg, 0, n);
+
```

Comment:
Few lines above there is ` ssl->in_offt = ssl->in_msg;`, are we clearing the same buffer twice?

Reply:
actually yes, good catch

Reply:
fixed